frontend/src/components/pages/HomePage.tsx

```diff
@@ -44,7 +44,8 @@ const HomePage = () => {
   });
   return (
     <Box>
-      <Header currentPageTitle="Add My Language" />
+      <Header title="" />
```

Comment:
nit: in Header, make the title default prop an empty string. that way this line can just be `<Header />`